data/human/human missions.txt

```diff
@@ -3601,3 +3601,76 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+
+
+
+mission "Repulsor failure"
+	landing
```

Comment:
add a clearance so that when you land again if this is a restricted planet you don't have to pay the credits twice, I think the authorities will understand you had a malfunction and did not actually land

Reply:
Good point, I will add clearance.